package/base-files/files/sbin/sysupgrade

```diff
@@ -14,7 +14,7 @@ export SAVE_OVERLAY=0
 export SAVE_OVERLAY_PATH=
 export SAVE_PARTITIONS=1
 export SAVE_INSTALLED_PKGS=0
-export SKIP_UNCHANGED=0
+export SKIP_UNCHANGED=
```

Comment:
SKIP_UNCHANGED needs to be unset to add args with ${SKIP_UNCHANGED:+arg1 arg2...}